assets/src/styles/blocks/core-overrides/Image.scss

```diff
@@ -30,12 +30,14 @@
   margin-top: $sp-1;
   margin-bottom: $sp-2;
 
-  &.is-style-rounded-180 {
-    @include rounded-image-size(180px);
-  }
+  &.is-style-rounded {
```

Comment:
I agree by doing this but take in mind it's incrementing the bundle size in 7 bytes (because it's a small change)

**Current implementation**
```sh
               Asset       Size  Chunks             Chunk Names
    ./bundle.min.css  545 bytes       0  [emitted]  bundle
./bundle.min.css.map  930 bytes          [emitted]  
    bundle.deps.json   15 bytes       0  [emitted]  bundle
Entrypoint bundle = ./bundle.min.css bundle.deps.json
[0] ./assets/src/scss/bundle.scss 39 bytes {0} [built]
```

**New implementation**
```sh
               Asset       Size  Chunks             Chunk Names
    ./bundle.min.css  545 bytes       0  [emitted]  bundle
./bundle.min.css.map  937 bytes          [emitted]  
    bundle.deps.json   15 bytes       0  [emitted]  bundle
Entrypoint bundle = ./bundle.min.css bundle.deps.json
[0] ./assets/src/scss/bundle.scss 39 bytes {0} [built]
```

Reply:
Ah okay, no problem I can revert it to how it used to be, it felt like less duplication this way but it's definitely not a big deal 🙂